tools/ci_build/github/azure-pipelines/win-wasm-ci-pipeline.yml

```diff
@@ -1,5 +1,24 @@
 jobs:
+- job: Extract_commit
+  pool: 'Win-CPU-2019'
+  timeoutInMinutes: 30
+  workspace:
+    clean: all
+  steps:
+  - checkout: none
+    fetchDepth: 1
+    submodules: false
+  - script: |
+     echo.$(Build.SourceVersion)
```

Comment:
just curious, does this mean that $(Build.SourceVersion) can change within a single build?

Reply:
if it is a PR build, yes.

each job in the pipeline will checkout `ref/pull/<PR_ID>/merge`. and this may be different if the master is updated.